docker-compose.production.yml

```diff
@@ -82,6 +82,20 @@ services:
     secrets:
       - petabox_seed
 
+  solr_updater:
+    # restart: always
+    hostname: "$HOSTNAME"
+    environment:
+      - OL_CONFIG=/olsystem/etc/openlibrary.yml
+      - SOLR_UPDATER_STATE_FILE=/1/var/run/openlibrary/solr-update.offset
```

Comment:
Hmmm... where to put this file... We do want it to persist between runs, so it can't live inside the container.  Let's place it in its own volume, and mount it at `/solr-update.offset`.